### PR TITLE
Added link for Flarum/core and Discord Chat

### PR DESCRIPTION
**Pull request by gurjyot**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -20,9 +20,13 @@ composer create-project flarum/flarum . --stability=beta
 
 Read the [Installation Guide](http://flarum.org/docs/installation) for more information.
 
+## Note
+
+[flarum/flarum](https://github.com/flarum/flarum) is a "skeleton" application which uses Composer to download [flarum/core](https://github.com/flarum/core) and main development goes in [flarum/core](https://github.com/flarum/core).
+
```
Comment (franzliedke) [inline readme.md]: 1. The developer has to use Composer, the application barely knows about it.
2. Let's split this into two sentences, the second being "Most development happens in flarum/core".

Comment (gurjyot) [reply readme.md]: I am unable to understand what exact sentence you want me to write in the first case.

Comment (franzliedke) [reply readme.md]: Something like "... is a skeleton application that bundles all dependencies such as flarum/core using Composer"?

Comment (franzliedke) [reply readme.md]: As long as it doesn't sound like flarum/flarum does any magic with Composer.

Comment (franzliedke) [reply readme.md]: > flarum/flarum is a skeleton application. Its dependencies, such as flarum/core (where most development happens), have to be installed using Composer.

Link where appropriate. :smile:

Comment (gurjyot) [reply readme.md]: To be honest, the first line has no value here. The second line holds most of the value here as many people search Flarum on Google or on Github and flarum/flarum is shown up first. They see that not much activity (commits) are going here so they think that the project is dead. By telling them - main development is going in Flarum/core we have point some of them to the right place. 
So basically, the first line is here just to support the second line, thus anything you say to write is good for me.

Comment (franzliedke) [reply readme.md]: Cool. Go with my last proposal then, please. :)

**Pull request merged**